4_Zadania_dodatkowe/4.ProgressBar/js/app.js

```diff
@@ -14,17 +14,12 @@ $(document).ready(function () {
         var clickedBarId = $(this).data('bar');
         // console.log(clickedBarWidth);
 
-        // barProgress.each(function(){
-        //     if (barProgress.attr("id") === clickedBarId) {
-        //     console.log(clickedBarId);
-        //     }
-        // });
+        barProgress
+            .filter('#'+clickedBarId)
+            .removeClass(['blue', 'orange', 'green'])
+            .addClass(clickedBarColor)
+            .animate({width: clickedBarWidth});
```

Comment:
one may say that creating all these `clickedBar...` variables is not needed as well
they are used only once so you could also write `.addClass($(this).data('color'))` and so on